api/main.py

```diff
@@ -2,7 +2,7 @@
 import flask
 import context
 from library.gmail import Gmail, GmailLogic
-from library.llm import LLM, Wizard
+from library.llm import LLM, Wizard, Hermes, Falcon, Mini, MistralInstruct, MistralOrca
```

Comment:
I added the capability to try multiple models. We'll likely want to select these at the API level but this PR is already too meaty. RIght now the Mistral models seem to do best at leveraging the context